samcli/local/docker/utils.py

```diff
@@ -154,3 +156,14 @@ def get_docker_platform(architecture: str) -> str:
     validate_architecture(architecture)
 
     return f"linux/{get_image_arch(architecture)}"
+
+
+def is_selinux_enabled():
+    """Return True if SELinux is enabled, False otherwise."""
+    if sys.platform == "linux":
+        if os.path.isfile("/usr/sbin/sestatus") and "enabled" in subprocess.Popen(
+            "/usr/sbin/sestatus", stdout=subprocess.PIPE
+        ).stdout.readline().decode("UTF-8"):
+            return True
+        else:
+            return False
```

Comment:
nit: Return the condition directly instead of returning True and False.
```suggestion
        return os.path.isfile("/usr/sbin/sestatus") and "enabled" in subprocess.Popen(
            "/usr/sbin/sestatus", stdout=subprocess.PIPE
        ).stdout.readline().decode("UTF-8")
```

Reply:
Thanks, I didn't run the pylint checks locally before the commit earlier. That is fixed with this most recent commit.